homeassistant/components/vacuum/neato.py

```diff
@@ -181,38 +174,26 @@ def state_attributes(self):
 
         return data
 
-    def turn_on(self, **kwargs):
+    def start(self):
         """Turn the vacuum on and start cleaning."""
+        self._clean_state = STATE_CLEANING
```

Comment:
`update` will be called directly after the method call when handling the service call, so it's better to let `update` handle the state update.